tests/baselines/reference/tsxReactEmitEntities.js

```diff
@@ -23,7 +23,9 @@ declare var React: any;
 <div>&#x1F408;&#x1F415;&#128007;&#128017;</div>;
 
 //// [file.js]
-React.createElement("div", null, "Dot goes here: \u00B7 &notAnEntity; ");
+React.createElement("div", null,
+    "Dot goes here: \u00B7 &notAnEntity;",
+    " ");

```

Comment:
Changing the arity of `createElement` here is going to be a runtime break. This needs to be preserved.

Reply:
Based on React's documentation this would not create an issue. https://reactjs.org/docs/react-api.html#createelement

Nonetheless, I do agree that the syntax is weird. In my small set of tests, I found no difference in adding the empty line in a single string than as an extra parameter.

Reply:
https://codepen.io/SeaRyanC/pen/eYBZOLJ

It's an observable runtime change that could break someone's app
```js
function myComp(props) {
  if (typeof props.children === "string") {
    return <div>Good</div>
  } else {
    return <div>Bad</div>
  }
}

ReactDOM.render(React.createElement(myComp, {}, "one "), document.getElementById("good"));
ReactDOM.render(React.createElement(myComp, {}, "one", " "), document.getElementById("bad"));

```

Reply:
I see. It makes sense, working on this then.